src/client/_clientwindow.py

```diff
@@ -1062,6 +1045,11 @@ def _tabChanged(self, tab, curr, prev):
     def mainTabChanged(self, curr):
         self._tabChanged(self.mainTabs, curr, self._main_tab)
         self._main_tab = curr
+        # the tab change works after client has initialized
+        if not self.init_ready:
+            self.init_ready = self._main_tab == 1  # chatTab
```

Comment:
I think it's best to just set it to True here, not everyone switches to chat tab at client start and it'll be confusing if announcements only show once you switch to the chat tab.

Reply:
Oh right, you mean the popup+sound?  I have that disabled ..forever, so forgot about those